color.js

```diff
@@ -0,0 +1,46 @@
+const currentColor = document.querySelector('#currentColor')
+const body = document.body
+
+const getHex = () => {
+    let color = '#'
+    let primaryColorNumber = 3
+    let randomRGBNumber
+    for( let i= 1;i<=primaryColorNumber;i++){
+        randomRGBNumber = Math.round(Math.random()*255)
+        if(randomRGBNumber.length==1){
```

Comment:
This condition does not work and leads to 5 character hex code creation.
randomRGBNumber is number and must not be checked by length property. Rewrite these condition